packages/kit/postinstall.js

```diff
@@ -7,32 +7,34 @@ import * as sync from './src/core/sync/sync.js';
 const cwd = process.env.INIT_CWD ?? process.cwd();
 process.chdir(cwd);
 
-const pkg = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+if (fs.readFileSync('package.json')) {
```

Comment:
```suggestion
if (fs.existsSync('package.json')) {
```
Also, to save some indentation, maybe do `if (!existsSync) return`?

Reply:
oops, good catch. I don't think you can early return from a module though?

Reply:
yeah, just checked — 'Illegal return statement'

Reply:
Oh right, forgot that this is top-level 😅